### ISSUE TITLE
cp: can't stat '/datadog-init/dd-java-agent.jar': No such file or directory

### ISSUE THREAD

**Issue by rdimash** [DataDog/dd-trace-java#7398]
We are using the latest image "public.ecr.aws/datadog/dd-lib-java-init:latest" to initialize the datadog agent of our deployments.
By approximately an hour we are faceing the following error:

"cp: can't stat '/datadog-init/dd-java-agent.jar': No such file or directory"

Currently, our pods cannot be restarted correctly, impacting all environments.
Can you resolve it?

Cheers,
Riccardo

**Comment by vandonr** [comment]
What exactly are you doing with that image ?
If you use the provided `copy-lib.sh` script, everything should still work fine.

The jar that was at the root is now in a `package` subfolder, I'll check with the team why this was done.

```
> docker run -it public.ecr.aws/datadog/dd-lib-java-init:1.38
/datadog-init $ ls
copy-lib.sh  package
/datadog-init $ ls package/
dd-java-agent.jar  version
```
vs
```
> docker run -it public.ecr.aws/datadog/dd-lib-java-init:v1.37
/datadog-init $ ls
copy-lib.sh        dd-java-agent.jar
```

**Comment by vandonr** [comment]
So, TL;DR: the internal structure of files in this container should be viewed as implementation detail. Those are meant to be used with the admission controller, as described here: https://docs.datadoghq.com/tracing/trace_collection/library_injection_local

We also realize that this is not something that was made clear where the container image is accessible, we're going to work on improving that.

In the meantime, I'm still interested in knowing how your setup uses that image, so that maybe I can redirect you to a way that is officially supported.

Note that you can also open a proper support request with datadog if you want to be able to exchange some details privately.
